src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -24,6 +24,7 @@ class Config:
         extra = "forbid" # Raise error on unknown fields
 
     nonTensorFields: List = Field(default_factory=list)
+    tensorFields: List = Field(default_factory=list)
```

Comment:
Why is this not `Field(default_factory=None)` just like what you did for `AddDocsParams`?

Reply:
Thanks. Fixed